assets/_scss/vk-post-author.scss

```
@@ -89,6 +89,13 @@ ul.sns_icons {
 		display: inline;
 		font-size: 28px;
 		margin-right: 10px;
+		// Reset the theme's global `i { margin-right: .4em; }` (e.g. Lightning), which stacks
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)